lib/Mojolicious/Controller.pm

```diff
@@ -11,6 +11,8 @@ use Mojo::Util;
 use Mojolicious::Routes::Match;
 use Scalar::Util ();
 
+use constant ABSOLUTE => qr!^(?:[^:/?#]+:|//|#)!;
```

Comment:
Make this an uppercase variable instead of a constant, then it is not a potential namespace pollution problem for subclasses.